www/src/pages/insights.jsx

```diff
@@ -216,3 +299,23 @@ InsightsPage.propTypes = {
     tab: PropTypes.oneOf(Object.values(INSIGHTS_TABS)),
   }).isRequired,
 };
+
+ComponentUsage.propTypes = {
+  name: PropTypes.string.isRequired,
+  componentUsageInProjects: PropTypes.arrayOf(PropTypes.shape({
+    name: PropTypes.string,
+    folderName: PropTypes.string,
+    version: PropTypes.string,
+    repositoryUrl: PropTypes.string,
+    componentUsageCount: PropTypes.number,
+    usages: PropTypes.arrayOf(PropTypes.shape({})),
```

Comment:
Can we expand on what this `PropTypes.shape({ ... })` should contain?

Reply:
Sure, updated